deploy/qdrant-cluster/templates/_helpers.tpl

```diff
@@ -60,3 +60,17 @@ Create the name of the service account to use
 {{- define "qdrant.serviceAccountName" -}}
 {{- default (printf "kb-%s" (include "clustername" .)) .Values.serviceAccount.name }}
 {{- end }}
+
+
+{{/*
+Define replicas.
+standalone mode: 1
+raftGroup mode: max(replicas, 3)
+*/}}
+{{- define "qdrant.replicas" }}
+{{- if eq .Values.mode "standalone" }}
+{{- 1 }}
+{{- else if eq .Values.mode "raftGroup" }}
```

Comment:
Does this cluster have raftGroup mode?

Reply:
No, we don't support raft protocol in qdrant. I will remove it.